scripts/actions_local_runner.py

```diff
@@ -0,0 +1,138 @@
+#!/bin/python3
+
+import subprocess
+import os
+import argparse
+import yaml
+import pprint
+import asyncio
+
+
+REPO_ROOT = os.path.dirname(os.path.dirname(__file__))
+
+
+class col:
+    HEADER = "\033[95m"
+    BLUE = "\033[94m"
+    GREEN = "\033[92m"
+    YELLOW = "\033[93m"
+    RED = "\033[91m"
+    RESET = "\033[0m"
+    BOLD = "\033[1m"
+    UNDERLINE = "\033[4m"
+
+
+def color(the_color, text):
+    return col.BOLD + the_color + str(text) + col.RESET
+
+
+def cprint(the_color, text):
+    print(color(the_color, text))
+
+
+async def run_step(step, job_name):
+    env = os.environ.copy()
+    env["GITHUB_WORKSPACE"] = "/tmp"
+    env["BASE_SHA"] = "123"
```

Comment:
Could you explain why this works?

Reply:
It doesn't, this line was leftover from trying to add `clang-tidy` since it already does the diff between master and HEAD to determine the files to look at, so it needs a base revision (this is kind of a problem since GitHub has an accurate base revision but we don't have that locally, `origin/master` will probably work most of the time but will probably have issues with rebases/ghstack)

Reply:
Ok gotcha; could you add a code comment explaining that?